woocommerce/woocommerce-ios
# Check for when Stripe is installed but not active

Closes: #5934

### Description
- If WCPay is not installed, and Stripe is, but isn't activated, prompt the merchant to activate the plugin.

### Nuances
- I extended `CardPresentPaymentsPlugins` with `image` so that all these onboarding views could just use it as the source for images. This improves upon what was done in #5930 and keeps things DRYer.
- I added unit tests for `test_onboarding_returns_stripe_plugin_not_activated_when_stripe_installed_but_not_active` but also `test_onboarding_returns_select_plugin_when_both_stripe_and_wcpay_plugins_are_active`

### Testing instructions
- Direct the app to a site with Stripe installed (but not active) and WITHOUT WCPay installed
- Enter Menu > Settings (gear) > In-Person Payments
- Ensure you are prompted to activate the plugin
- New unit tests were added. Ensure they pass as well

### Screenshots

<img src="https://user-images.githubusercontent.com/1595739/150431646-4fd41dfe-2092-42ca-8110-d67abf193311.jpg" width=50% />

---
